packages/analytics/analytics-chart/src/locales/en.json

```diff
@@ -23,7 +23,9 @@
     "exportDescription": "Exports a CSV of the data represented in the chart.",
     "previewRows": "Previewing {rowsMax} of {rowsTotal} row{plural}",
     "downloadButton": "Export",
-    "cancelButton": "Cancel"
+    "cancelButton": "Cancel",
+    "UtcOffset": "UTC Offset",
+    "Timestamp": "Timestamp"
   },
   "legend": {
     "datapointValueDisplay": "{value} {unit}"
```

Comment:
I think you can get rid of the old translations (PascalCase and SCREAMING_SNAKE_CASE, below).  If they still surface under any circumstances, we should figure that out sooner rather than later.